src/asmcnc/apps/maintenance_app/screen_maintenance.py

```diff
@@ -538,8 +554,29 @@ def __init__(self, **kwargs):
         self.z_lubrication_reminder_widget = widget_maintenance_z_lubrication_reminder.ZLubricationReminderWidget(machine=self.m, screen_manager=self.sm, localization=self.l)
         self.z_lubrication_reminder_container.add_widget(self.z_lubrication_reminder_widget)
 
+        # OPTIONAL + TAB
+
+        ## Enable tab
+
+        if not self.m.theateam() or True:
+            self.update_strings()
+            return
```

Comment:
I think the better way of doing this would be to invert the condition and nest all of the code below, aside from update_strings. This way the update_strings function doesn't need to be duplicated, and I think using return should be avoided in case you want to add code below this and forget about it